addons/packages/vsphere-csi/2.4.1/bundle/config/overlays/update-strategy-overlay.yaml

```diff
@@ -0,0 +1,50 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#! We are adding this overlay in the package to accomandate the need from vSphere supervisor cluster:
+#! `deployment.spec.strategy.type` is configured to `RollingUpdate`
+#! `deployment.spec.strategy.rollingUpdate.maxUnavailable` is set to `0`.
```

Comment:
nitpick: we aren't enforcing these numbers with overlays so the comments are confusing.

Reply:
Comment states `#! This overlay makes configuring the above parameters possible`, will refactor the comments for all packages in a follow up MR.